scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -218,6 +230,33 @@ export class DependencyLinker {
     return result;
   }
 
+  _getLinksToPeers(
+    componentDirectoryMap: ComponentMap<string>,
+    options: {
+      finalRootDir: string;
+      linkToDir: string;
+    }
+  ): LinkToDirResult[] {
+    const peers = new Set<string>();
+    componentDirectoryMap.toArray().forEach(([component]) => {
+      for (const peer of Object.keys(component.state._consumer.peerPackageDependencies)) {
```

Comment:
Use the new api for this.
Something like depsResolver getDependencies(component).byType